app-frontend/src/app/components/map/nodeSelector/nodeSelector.component.js

```diff
@@ -0,0 +1,14 @@
+import nodeSelectorTpl from './nodeSelector.html';
+const nodeSelector = {
+    templateUrl: nodeSelectorTpl,
+    controller: 'NodeSelectorController',
+    bindings: {
+        nodeMap: '<',
+        side: '@',
```

Comment:
this probably isn't the best way to do things - i'm gonna try to think of a better way